styles.css

```diff
@@ -49,155 +31,376 @@ body {
   font-weight: 700;
   font-size: 18px;
   line-height: 20px;
+  color: #6070ff;
+}
+
+.hamburger-icon {
   display: flex;
-  align-items: center;
-  text-align: center;
+  justify-content: flex-end;
   color: #6070ff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
-
-.icon {
-  height: 0;
-  width: 16.000000000000075px;
-  left: 4px;
-  top: 12px;
-  border-radius: 0;
-  margin-left: 102px;
-  display: inline-flex;
+}
+
+.fa-bars {
   color: #6070ff;
 }
 
-.text,
-.description {
-  width: 300px;
-  margin: 0;
+.f-ec {
+  display: flex;
+  justify-content: space-between;
 }
 
-.text {
-  margin: 30px 0 0 0;
-  height: 150px;
+.landing-page-main-head {
   font-family: "Poppins", sans-serif !important;
```

Comment:
- [Optional] Please kindly note that it is best practice not to overuse the `!important` rule in CSS. Kindly correct this, please. Check out the CSS best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).